Project/VetAppointmentAPI/VetAppointmentAPI/VetAppointment.Infrastructure/Repositories/Impl/OfficeRepository.cs

```diff
@@ -8,5 +8,11 @@ namespace VetAppointment.Application.Repositories.Impl
     public class OfficeRepository : BaseRepository<Office>, IOfficeRepository
     {
         public OfficeRepository(IDatabaseContext context) : base(context) { }
+
+        public async Task<Office> UpdateAsync(Office office)
```

Comment:
Let's move this method to the IBaseRepository interface and BaseRepository abstract implementation

Reply:
there is no need to move. we only have 2 controllers using MediatR, if we modify Update method we should refactor all controllers. this action can be perform in the next iteration: modify controllers and methods from repository